src/qat/purr/backends/qblox/device.py

```diff
@@ -285,7 +290,11 @@ def start_playback(self, repetitions: int, repetition_time: float):
                         ) / sequencer.integration_length_acq()
 
                     sequencer.delete_acquisition_data(all=True)
-                    sequencer.sync_en(False)
```

Comment:
Here you're replacing a previous call with one in a separate iteration but there was no previous `sequencer.sync_en(True)` call. Could you explain what this does?

Reply:
previously sync was being set as part of the configuration step. Not any more, everything must remain false, except for the involved module sequencer in the experiment